core/src/avm2/globals/flash/display/NativeMenu.as

```diff
@@ -1,6 +1,141 @@
-package flash.display {
+package flash.display
+{
+
     import flash.events.EventDispatcher;
-    public class NativeMenu extends EventDispatcher {
+    import __ruffle__.stub_method;
+    import __ruffle__.stub_getter;
+    
+    [API("661")]
+    public class NativeMenu extends EventDispatcher
+    {
+
+        // Indicates whether any form of native menu is supported on the client system.
+        private var _isSupported:Boolean;
+
+        // The array of NativeMenuItem objects in this menu.
+        public var items:Array;
+
+        // The number of NativeMenuItem objects in this menu.
+        private var _numItems:int;
```

Comment:
I feel like instead of a separate field, the actual length of `items` could be returned from `get numItems`. That way, there's no possibility of inconsistency. And this member can't be set in any way anyway.